### ISSUE TITLE
Opam short download link

### ISSUE THREAD

**Issue by rjbou** [ocaml/ocaml.org#2728]
We'd like to have a short link for opam install script. For the moment the install instruction are:
```
bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh) --version 2.2.1"
```
We're thinking of adding `opam.ocaml.org/install.sh`. Do you have plans for an installer for ocaml.org?

/cc @kit-ty-kate @dra27

**Comment by cuihtlauac** [comment]
@rjbou, do you mean this:
```
bash -c "sh <(curl -fsSL https://ocaml.org/install.sh) --version 2.2.1"
```

**Comment by kit-ty-kate** [comment]
something like that yes

**Comment by sabine** [comment]
I don't see a need for additionally having `https://ocaml.org/install.sh` (this could in fact lead to conflicts in the future, as it's not referring to which tool is being installed).

Providing `https://opam.ocaml.org/install.sh` sounds great to me. Another workable alternative would be `https://ocaml.org/install-opam.sh`.

**Comment by kit-ty-kate** [comment]
> Providing `https://opam.ocaml.org/install.sh` sounds great to me.

that was done in https://github.com/ocaml-opam/opam2web/pull/241 btw. We'll update our documentation to reflect that. I'll also send a PR to update ocaml.org in a minute

**Comment by rjbou** [comment]
Of course, I didn't mean `https://ocaml.org/install.sh` directly, as it is to install opam, not OCaml itself ^^. Under `ocaml.org` link, it would be as proposed `https://ocaml.org/install-opam.sh`, or `https://ocaml.org/install/opam.sh` if there is a plan to have several tool download links on ocaml.org for example.

**Comment by cuihtlauac** [comment]
Fixed by #2741 ?
